dropwizard-jsonapi/src/main/java/io/rtr/jsonapi/filter/JsonApiResponseFilter.java

```diff
@@ -213,4 +216,11 @@ private boolean isApplicable(ContainerRequestContext requestContext) {
 				.stream()
 				.anyMatch(m -> JSONAPI_MEDIATYPE.equals(m));
 	}
+
+	@VisibleForTesting
```

Comment:
Ah I see.  I was trying to directly test the filter method at first, but I was having trouble creating a mock for the injected fields (UriInfo, ResourceMappingContext).  Do u know how to do that?